packages/error/src/index.ts

```diff
@@ -0,0 +1,21 @@
+const isProduction: boolean = process.env.NODE_ENV === "production";
+const prefix = "\nVisit https://community.frontity.org for help! 🙂\n";
+
+// Throw an error if the condition fails
+// Strip out error messages for production
+// > Not providing an inline default argument for message as the result is smaller
+export default function invariant(condition: boolean, message?: string) {
```

Comment:
As it looks like we are going to have two exports (`invariant` and `warning`), what about making them named exports?

That way we can also ensure that `babel-plugin-dev-expression` always works because it seems like it only works when it finds `invariant` and `warning` functions, and it doesn't work for other names like `error` or `warn`.

Reply:
yes, that's what I was thinking as well.

Reply:
Actually, I removed the babel plugin for now. I realized that we probably won't be using the `invariant` function very much. If I'm wrong, we can always add it back in 🙂 .